frontend/src/app/testResults/TestResultsList.tsx

```diff
@@ -100,21 +100,22 @@ function testResultRows(
         action: () => setEmailModalTestResultId(r.internalId),
       });
     }
-    actionItems.push({
-      name: "View details",
-      action: () => setDetailsModalId(r.internalId),
-    });
     actionItems.push({
       name: "Text result",
       action: () => setTextModalId(r.internalId),
     });
+
     const removed = r.correctionStatus === "REMOVED";
     if (!removed) {
       actionItems.push({
         name: "Mark as error",
         action: () => setMarkErrorId(r.internalId),
       });
     }
+    actionItems.push({
+      name: "View details",
+      action: () => setDetailsModalId(r.internalId),
+    });
```

Comment:
was in a pairing session with @kenieh when they mentioned they wanted the order of these options changed, so I went ahead and did that in this PR.